### ISSUE TITLE
Version property public readable

### ISSUE THREAD

**Issue by dwight66** [GaretJax/phpbrowscap#57]
Can you make the 'const VERSION' property public accessible? I would like to use your phpbrowscap as an unmodified library into my code. Making this property public enables me to test if the minimum version of phpbrowscap is being used.

**Comment by quentin389** [comment]
It's a constant. It IS publicly readable, as any other constant in PHP.

http://www.php.net/manual/pl/language.oop5.constants.php

**Comment by dwight66** [comment]
Thank you for your reaction.
You are right. (of course, you wrote it :-) ) I found a way to get its
version number. It was not there where I expected it. (Lack to php
experience, I presume...). I'm used to instantiate a class and then read a
porperty from the instance. However in the instance I couldn't find the
version property, so I suspected the property to be hidden. (The property
has not an access modifier, so I presumed it was 'private'.) By reading it
directly from the class with 'Browscap::VERSION' I was able to get the
version.
Thank you also for creating browscap. I hope you 'll keep it up for a long
time.
